android/assets/jsons/translations/template.properties

```diff
@@ -1034,6 +1045,7 @@ Must be on [terrain] =
 Can only be built on [tileFilter] tiles = 
 Cannot be built on [tileFilter] tiles = 
 Does not need removal of [feature] = 
+Gain a free [building] [cityFilter] = 
```

Comment:
This too already exists.

Reply:
In code yes, but try template.properties Ctrl-F "Gain a free" -> not found. That was my reason to include, not because I introduced the text.